### PR TITLE
Argparse

### PR DESCRIPTION
**Pull request by Sasafrass**
Implements:
* Argparse parameters for train_wrapper.py ("cpu" is default now for device).
  * For filename_datasets it now accepts a tab separated list.
* Allow the run function to accept these arguments.
* Update to requirements.txt for tqdm and black.

### CODE REVIEW

File: app/ml_models/rnn/train.py [hunk 1]
```diff
@@ -101,7 +101,6 @@ def train(
                     ),
                 )
 
-            exit()
```
Comment (Sasafrass) [inline app/ml_models/rnn/train.py]: agreed

**Comment by annaproxy** [comment]
Added some more args for training, 

- vocabulary file
- batch size
- epochs or training steps
- save_every
- learning rate, momentum, hidden size

Further small improvements:
- Don't build vocab if file given exists
- Create directory "models" if it does not exist
- Vocab can be built from multiple files
- Some renaming (num_epochs, model_name, etc
- Refactor load_model so the torch "core" loader can be reused

**Review by annaproxy — APPROVED** [review]

File: app/ml_models/rnn/train.py [hunk 2]
```diff
@@ -53,7 +82,7 @@ def train(
             # output, _  = rnn(input_line_tensor)
             # loss = criterion(output.permute(1, 2, 0), target_line_tensor.permute(1,0))
 
-            # Run model ye new way
+            # Run model ye newe way
```
Comment (Sasafrass) [inline app/ml_models/rnn/train.py:85]: important

File: train_wrapper.py [hunk 3]
```diff
@@ -1,34 +1,157 @@
...
 
 from app.ml_models.rnn.data_tools import WordLevelDataset
 from app.ml_models.rnn.vocabulary import Vocabulary
 from app.ml_models.rnn.train import train
+from app.ml_models.rnn.loaded_rnn_model import init_torch_model_from_path
 from app.ml_models.rnn.rnn_model import RNNAnna
 
 
-def run():
+def _run(**kwargs):
```
Comment (Sasafrass) [inline train_wrapper.py:17]: I like the addition of the leading underscore. Should we also become more strict with the return types on all of our functions?
```suggestion
def _run(**kwargs) -> None:
```
Although we're gonna be adding a bunch of "-> None"s
